code/game/objects/items/devices/education_chips.dm

```diff
@@ -0,0 +1,95 @@
+//This is for electronic versions of learning, for now solely martial arts
+
+/obj/item/clothing/head/AED
+  name = "Automatic Education Device"
+  desc = "A complex assortment of metal, electrodes, and wires. Simply put the headset on, insert an education chip, and turn it on to instantly learn the contents of said chip."
+  icon_state = "electrode_helmet"
+  actions_types = list(/datum/action/item_action/instant_learn)
+  pocket_storage_component_path = /datum/component/storage/concrete/pockets/tiny/AED
+
+/obj/item/clothing/head/AED/item_action_slot_check(slot)
+  if(slot == SLOT_HEAD)
+    return 1
+
+/obj/item/clothing/head/AED/ui_action_click(mob/user, action)
+  educate()
+
+/obj/item/clothing/head/AED/verb/educate()
+  set src in usr
+  if(!src.contents.len)
+    to_chat(usr, "<span class='warning'>There is nothing loaded in the [src]!</span>")
+    return
+  for(var/obj/item/education_chip/E in src.contents)
+    if(!E.subject)
+      to_chat(usr, "<span class='warning'>The Education Chip doesn't have a subject!</span>")
+      return
```

Comment:
if you're gonna make this a for loop these should be continues. pockets can hold two chips, randomly ending sometimes depending on the order you added the chips is bad.

Reply:
It can only hold one chip